proto/nvme_gw.proto

```diff
@@ -103,20 +109,27 @@ message nsid {
 	int32 nsid = 1; //nsid created
 }
 
+message ns_delete_req {
+	string subsystem_nqn = 1;
+	int32 nsid = 2;
```

Comment:
Why is `optional string tgt_name` not added here as well?  Honestly, I don't see the point of adding these unused fields in advance but `nvmf_subsystem_remove_ns` accepts it and you added them for `host_delete_req` and `listener_delete_req`.